package.json

```
@@ -51,8 +51,8 @@
   },
   "peerDependencies": {
     "prop-types": ">=15.5.6",
-    "react": ">=15",
-    "react-dom": ">=15"
+    "react": "15.x || 16.x || 17.x",
+    "react-dom": "15.x || 16.x || 17.x"
```

Although I understand the reasoning behind explicitly calling out the React versions, I think this actually adds more of a headache, since if React releases a new version, we won't be compatible with it and will need to be quick on the ball here. I think we can still keep this change but add something like `>17` so that new versions are automatically supported. e.g. 

```suggestion
    "react": "15 || 16 || 17 || >17",
    "react-dom": "15 || 16 || 17 || >17"
```